deployment/helm/node-feature-discovery/values.yaml

```diff
@@ -476,8 +480,10 @@ topologyUpdater:
 ### <NFD-TOPOLOGY-UPDATER-CONF-END-DO-NOT-REMOVE>
 
   enable: false
-  args: []
   createCRDs: false
+  args: []
+  envs: []
```

Comment:
Looking at this now, I'd prefer name `extraEnvs`.

I know `envs` is in line with `args`. I recall that I have myself added the `args` and I don't like the name :roll_eyes:. I'd suggest I rename that to `extraArgs` (in a separate PR) and you rename to `extraEnvs` in this PR.

WDYT?

Nevertheless,we need to document the new values in `docs/deployment/helm.md`